inc/template.php

```diff
@@ -352,6 +351,20 @@ function tpl_metaheaders($alt = true) {
     return true;
 }
 
+function _tpl_ensureJSINFO() {
```

Comment:
I'm not sure if this should be in `inc/template.php` and I'm also not so keen about the name. Maybe `initJSInfo` or `upgradeJSInfo`?